Apps/Common/src/Messaging/DbOutboxStore.cs

```diff
@@ -24,31 +24,30 @@ namespace HealthGateway.Common.Messaging;
 using System.Threading.Tasks;
 using Hangfire;
 using HealthGateway.Common.Utils;
-using HealthGateway.Database.Context;
+using HealthGateway.Database.Delegates;
 using HealthGateway.Database.Models;
-using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Logging;
 
 /// <summary>
 /// Outbox store implementation that uses a dedicated Outbox table in the database.
 /// </summary>
 internal class DbOutboxStore : IOutboxStore
 {
-    private readonly GatewayDbContext dbContext;
+    private readonly IOutboxDelegate outboxDelegate;
     private readonly IBackgroundJobClient backgroundJobClient;
     private readonly IMessageSender messageSender;
     private readonly ILogger<DbOutboxStore> logger;
 
     /// <summary>
-    /// Initializes a new instance of the <see cref="DbOutboxStore"/> class.
+    /// Initializes a new instance of the <see cref="DbOutboxStore"/> class
     /// </summary>
-    /// <param name="dbContext">The EF database context.</param>
-    /// <param name="backgroundJobClient">Hangfire background job client.</param>
-    /// <param name="messageSender">The destination message sender to forward messages to.</param>
-    /// <param name="logger">A logger.</param>
-    public DbOutboxStore(GatewayDbContext dbContext, IBackgroundJobClient backgroundJobClient, IMessageSender messageSender, ILogger<DbOutboxStore> logger)
+    /// <param name="outboxDelegate">the outbox db delegate</param>
```

Comment:
```suggestion
    /// <param name="outboxDelegate">The outbox db delegate.</param>
```
There is probably a warning regarding current param documentation. Missing capital at beginning of sentence.  Missing period at end of sentence.